bots/zephyr_mirror_backend.py

```diff
@@ -717,8 +751,11 @@ class and your mirroring bot does not have access to the relevant \
 %s
 
 %s""" % (support_heading, stderr, support_closing))
+    return
```

Comment:
We need to the mentor who proposed that to review. I'm a little bit tired now and afraid that wont catch everything.